AAI/AAIConnectProfile.md

```diff
@@ -169,16 +169,16 @@ specific assignment within the organization that made the assertion.
 
 [GA4GH-Passport](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md) - DURI
 
-[RFC8693](https://www.rfc-editor.org/info/rfc8693)  Jones, M., Nadalin, A., Campbell, B., Ed., Bradley, J.,
+[RFC8693](https://www.rfc-editor.org/info/rfc8693) - Jones, M., Nadalin, A., Campbell, B., Ed., Bradley, J.,
```

Comment:
An alternative would be to delete *all* the separation hyphens from the spec list (but it should be consistent)